bluepysnap/frame_report.py

```diff
@@ -111,6 +112,59 @@ def get(self, group=None, t_start=None, t_stop=None):
         return res
 
 
+class FilteredFrameReport(object):
+    """Access to filtered FrameReport data."""
+    def __init__(self, frame_report, group=None, t_start=None, t_stop=None):
+        """Initialize a FilteredFrameReport.
+
+        A FilteredFrameReport is a lazy and cached object which contains the filtered data
+        from all the populations of a report.
+
+        Args:
+            frame_report (FrameReport): The FrameReport to filter.
+            group (None/int/list/np.array/dict): Get spikes filtered by group. See NodePopulation.
+            t_start (float): Include only frames occurring after this time.
+            t_stop (float): Include only frames occurring before this time.
+
+        Returns:
+            FilteredFrameReport: A FilteredFrameReport object.
+        """
+        self.frame_report = frame_report
+        self.group = group
+        self.t_start = t_start
+        self.t_stop = t_stop
+
+    @cached_property
+    def report(self):
+        """Access to the report data.
+
+        Returns:
+            pandas.DataFrame: A DataFrame containing the data from the report. Row's index is the
+            different timestamps and the column's MultiIndex is :
```

Comment:
Better to add an extra indentation on line continuations for args and return values

Reply:
ok.